docker-agent/builder.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+if [ "$RUN_DETEKT" = true ]; then
+   echo "Running detekt"
+   ./gradlew detektAll
```

Comment:
uyou can replace this with `./gradlew staticCodeAnalysis` it will give us more control, for reference, see the [AR-builder.groovy](https://github.com/wireapp/wire-android-reloaded/blob/fd1b0e2241bf1258f79062c9f032b4d6c6e78fc8/AR-builder.groovy#L224)